### DISCUSSION TITLE
Element query field - using the condition builder

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#13738] [Ideas] [answered]
It would be useful if we could use the condition builder as a field. 

Example usage:

Lets say that we have some kind of list page and want to decide which entries should be displayed there. But not by selecting specific entries using entry page, but by defining element query parameters for entry query. So, for example we can use section option of condition builder to decide that all entries from "blog" section should be included, and then add "id" condition saying that all except one with id 20 should be included.

**Comment by brandonkelly** [comment, marked as answer]
We’ve definitely thought about this idea! In the meantime, you can pull it off with the [Conditions](https://plugins.craftcms.com/conditions?craft4) plugin.

**Comment by piotrpog** [comment]
@brandonkelly 
Thanks for mentioning this plugin, i think it would be what im looking for.
